chcemvediet/apps/accounts/forms.py

```diff
@@ -65,15 +70,91 @@ def signup(self, request, user):
 class SettingsForm(forms.Form):
 
     anonymize_inforequests = forms.BooleanField(
-        label=_(u'accounts:SettingsForm:anonymize_inforequests:label'),
-        required=False,
-    )
+            label=_(u'accounts:SettingsForm:anonymize_inforequests:label'),
+            required=False,
+            )
+
+    custom_anonymization = forms.BooleanField(
+            label=_(u'accounts:SettingsForm:custom_anonymization:label'),
+            required=False,
+            widget=forms.CheckboxInput(attrs={
+                u'class': u'pln-toggle-changed',
+                u'data-container': u'form',
+                u'data-hide-target-true': u'.form-group:has(.chv-visible-if-custom-anonymization)',
+                u'data-disable-target-true': u'.chv-visible-if-custom-anonymization',
+            }),
+            )
+
+    help_text = ungettext(
+            u'accounts:SettingsForm:custom_anonymized_strings:help_text',
+            u'accounts:SettingsForm:custom_anonymized_strings:help_text %(count)s.',
+            WORD_SIZE_MIN) % {u'count': WORD_SIZE_MIN}
```

Comment:
@viliambalaz Este pozor, namiesto `ungettext` treba pouzit `ungettext_lazy`. Rozdiel je v tom, ze "lazy" funkcie sa nevyhodnotia hned pri zavolani, ale az pri pouziti ich vysledku. A zakazdym nanovo. Vid: https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#lazy-translation

Predstav si, ze mas SK a EN jazykove mutacie a EN je defaultna. V case, ked sa vyhodnocuje riadok `help_text = ungettext(...)` sa pouzije EN lokalizacia a dostanes EN string, ktory posles do `CharField`. Takze vzdy sa vsade bude pouzivat EN string bez ohladu na to, aku ma user aktualne nastavenu lokalizaciu. Ale ked pouzijes `ungettext_lazy`, tak realne sa ta funkcia nevyhodnoti, len sa vrati objekt, ktory sa vyhodnoti az pri pouziti. Takze do `CharField` posles ten objekt, ktory sa pri kazdom vyrendrovani nanovo vyhodnoti a pouzije aktualnu verziu prekladu podla aktualnej lokalizacie usera. 

Okrem toho treba aj doplnenie hodnoty `WORD_SIZE_MIN` za parameter `count` spravit "lazy". Aby si nedostal konkretny vyhodnoteny string, ale iba objekt, ktory sa "lazy" vyhodnoti az pri pouziti. Na taketo "lazy" formatovanie mame uz spravenu pomocnu funkciu `poleno.utils.lazy.lazy_format`. Pozor `lazy_format` pouziva "python-brace-format" nie `%`.

Cize nakoniec dostanes:
```
help_text=lazy_format(ungettext_lazy(
  u'accounts:SettingsForm:custom_anonymized_strings:help_text',
  u'accounts:SettingsForm:custom_anonymized_strings:help_text {count}',
  WORD_SIZE_MIN), count=WORD_SIZE_MIN)
```

Btw, pls samotny `help_text` nenechavaj ako class property, ale daj ho rovno na miesto pouzitia, nech v tej classe nevysia zbytocnosti.